src/components/Typography/Typography.stories.tsx

```diff
@@ -0,0 +1,147 @@
+      },
+    },
+  },
+} as ComponentMeta<typeof Typography>;
+
+const Template: ComponentStory<typeof Typography> = (args) => (
+  <Typography {...args} />
+);
+
+const MultipleTemplates: ComponentStory<typeof Typography> = (args) => (
+  <>
+    <Typography {...args} />
+    <Typography {...args} />
+  </>
+);
+
+export const Default = Template.bind({});
+Default.args = {
+  children: 'Hello world!',
+  variant: 'body2',
+};
+
+export const Paragraph = Template.bind({});
+Paragraph.args = {
+  children:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+  paragraph: true,
+};
+
+export const NoWrap = Template.bind({});
+NoWrap.args = {
+  children:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+  noWrap: true,
+};
+
+export const GutterBottom = MultipleTemplates.bind({});
+GutterBottom.args = {
+  children:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
+  gutterBottom: true,
+};
+
+export const Headings: ComponentStory<typeof Typography> = () => (
+  <>
+    <Typography variant="h1">H1</Typography>
+    <Typography variant="h2">H2</Typography>
+    <Typography variant="h3">H3</Typography>
+    <Typography variant="h4">H4</Typography>
+    <Typography variant="h5">H5</Typography>
+    <Typography variant="h6">H6</Typography>
+  </>
+);
+
+export const Subtitles: ComponentStory<typeof Typography> = () => (
+  <>
+    <Typography variant="subtitle1">Subtitle 1</Typography>
+    <Typography variant="subtitle2">Subtitle 2</Typography>
+  </>
+);
+
+export const BodySizes: ComponentStory<typeof Typography> = () => (
+  <>
+    <Typography variant="body1">Body 1</Typography>
+    <Typography variant="body2">Body 2</Typography>
+  </>
+);
+
+export const OtherSizes: ComponentStory<typeof Typography> = () => (
+  <Grid container flexDirection="column">
```

Comment:
Does this work without having any `<Grid item>`? Why not `<> ... </>` like in the story above?

Reply:
This OtherSizes for some reason they display in one line, without spaces. So decided to add some spacing, that it would looks more proper.